lib/repo.js

```diff
@@ -71,8 +77,8 @@ function repo (options, callback) {
       }
 
       var urls = [
-        'https://api.github.com/repos/' + user + '/' + repo + '/contents/' + path + '.geojson',
-        'https://raw.github.com/' + user + '/' + repo + '/' + branch + '/' + path + '.geojson'
+        'https://api.github.com/repos/' + user + '/' + repo + '/contents/' + path + extension,
+        'https://raw.github.com/' + user + '/' + repo + '/' + branch + '/' + path + extension
       ]
```

Comment:
agreed and fixed!  thanks for the catch.
i actually learned a lot working on this bug (mostly about how to `npm link` from one directory to another).